### ISSUE TITLE
support of query by all HGVSc

### ISSUE THREAD

**Issue by jjgao** [genome-nexus/genome-nexus#304]
This should be straightforward and we may be able to utilize VEP hgvsg API.

**Comment by zhx828** [comment]
@inodb @leexgh do you think this can be prioritized in the next couple month?

**Comment by leexgh** [comment]
@zhx828 Yes we could do that. @inodb Let's discuss it at GN planning meeting

**Comment by zhx828** [comment]
@leexgh @inodb hi guys, following up on this topic. Any plan to support this? Thanks!
